.circleci/config.yml

```diff
@@ -122,7 +122,7 @@ workflows:
           file: mapping/staging/mapping.txt
           app: $APP_CENTER_APP_NAME
           token: $APP_CENTER_TOKEN
-          version-name: echo $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d \''";\')
+          version-name: echo $(testapp/version-name-staging.txt)
```

Comment:
do you really need to create a file? If you included what you do line 45 into script, do you think it's possible?

Reply:
@Climbatize Yeah, the original purpose to open this PR to investigate an issue e.g. `No such file or directory` while trying to grep the value by traversing in the given file path. Because the existing script in `master` branch we have still producing empty `version-name` value in app-center.

So, this was my initial approach by storing the value in a temporary file e.g. `version-name-staging.txt` or in a temporary variable then use it directly here (L125). 

I think, `No such file or directory` is kinda misleading, means its finding the file but there is some other kinda issue.